homeassistant/components/plaato/const.py

```diff
@@ -1,3 +1,24 @@
 """Const for GPSLogger."""
 
 DOMAIN = "plaato"
+PLAATO_DEVICE_SENSORS = "sensors"
+PLAATO_DEVICE_ATTRS = "attrs"
+
+CONF_USE_WEBHOOK = "use_webhook"
+CONF_DEVICE_TYPE = "device_type"
+CONF_DEVICE_NAME = "device_name"
+CONF_CLOUDHOOK = "cloudhook"
+CONF_UPDATE_INTERVAL = "update_interval"
```

Comment:
Minor: Possibly could use use some of the common/homeassistant/const.py consts here rather than making new ones e,g, CONF_SCAN_INTERVAL